### ISSUE TITLE
Troubles installing the Helm chart on minikube / macOS

### ISSUE THREAD

**Issue by bartjkdp** [Amsterdam/openstad-kubernetes#4]
I am having troubles using the Helm chart on minikube / macOS:

```bash
helm dependency update
helm install openstad .
```

Results in infinite waiting....

This has to do with the pre-install job. When I remove the pre-install job, everything installs just fine.

Any suggestions how I could further debug the issue? During the pre-install job I do not see a container running.

**Comment by jaapgorjup** [comment]
@diederikgithub you have seen this before? Would this mean the pre-install job is waiting on a resource that still needs to be installed after the job?

**Comment by jaapgorjup** [comment]
@bartjkdp Is your Minikube standard or did you enable RBAC? The pre-install Job works based on a serviceaccount that needs to be available and can be set. On a standard Minikube RBAC seems to be disabled. Can you check?

**Comment by bartjkdp** [comment]
Sure, RBAC is enabled. It is a default minikube installation:

```bash
$ minikube start --cpus=2 --memory=8192 --disk-size=60G
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-21T14:51:23Z", GoVersion:"go1.14.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:48:36Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by diederikgithub** [comment]
@bartjkdp Did you create the namespace 'openstad' before starting the install, if not, could you?

**Comment by bartjkdp** [comment]
@diederikgithub this was the problem. As the hardcoded namespaces are now fixed in #7 and #3, install now works!
